oe_theme.theme

```diff
@@ -1045,3 +1045,41 @@ function oe_theme_preprocess_language_selection_page_content(&$variables) {
     $language['path'] = $language['href'];
   }
 }
+
+/**
+ * Implements template_preprocess_block__oe_theme_main_page_content().
+ */
+function oe_theme_preprocess_block__oe_theme_main_page_content(&$variables) {
+  $variables['attributes']['class'][] = 'ecl-u-mv-l';
+}
+
+/**
+ * Implements hook_preprocess_table().
+ */
+function oe_theme_preprocess_table(array &$variables): void {
+  $variables['zebra'] = 'true';
+
+  $name_col = [];
+  foreach ($variables['header'] as $key => &$cell) {
+    $cell['label'] = $cell['content'];
+
+    if ($cell['attributes']['colspan']) {
+      $cell['colspan'] = (string) $cell['attributes']['colspan'];
+    }
+
+    for ($i = 1; $i <= (array_key_exists('colspan', $cell) ? intval($cell['colspan']) : 1); $i++) {
+      $cell_label = (array) $cell['label'];
+      $name_col[] = array_key_exists('#markup', $cell_label) ? $cell_label['#markup'] : $cell['label'];
```

Comment:
Can we add a small comment as to why we are doing this? Something like: "The contents of a drupal cell can be a render array so we need to make sure we have an string to set as the 'data-ecl-table-header' property later"

Reply:
done.